From d00b69d601d09a23b6dca8c0fcc8200e97980a5b Mon Sep 17 00:00:00 2001
Message-Id: <d00b69d601d09a23b6dca8c0fcc8200e97980a5b.1675261799.git.nabijaczleweli@nabijaczleweli.xyz>
In-Reply-To: <727802bf80be4a3c817e52b5fc57868bcaf17999.1675261799.git.nabijaczleweli@nabijaczleweli.xyz>
References: <727802bf80be4a3c817e52b5fc57868bcaf17999.1675261799.git.nabijaczleweli@nabijaczleweli.xyz>
From: =?UTF-8?q?=D0=BD=D0=B0=D0=B1?= <nabijaczleweli@nabijaczleweli.xyz>
Date: Wed, 1 Feb 2023 15:20:25 +0100
Subject: [PATCH 2/2] d/p/initrd-not-initramfs.patch: rename to initrd.img
X-Mutt-PGP: OS

This has been a consistent source of confusion from the maintainer,
who updates the patch according to its filename, not according to the
message in the original commit 8ffe423532fa8950a422787d7f2acd11822b8015

Also, this matches the existing naming convention
---
 debian/patches/{initrd-not-initramfs.patch => initrd.img} | 0
 debian/patches/series                                     | 2 +-
 2 files changed, 1 insertion(+), 1 deletion(-)
 rename debian/patches/{initrd-not-initramfs.patch => initrd.img} (100%)

diff --git a/debian/patches/initrd-not-initramfs.patch b/debian/patches/initrd.img
similarity index 100%
rename from debian/patches/initrd-not-initramfs.patch
rename to debian/patches/initrd.img
diff --git a/debian/patches/series b/debian/patches/series
index 19828f40..b24bed0e 100644
--- a/debian/patches/series
+++ b/debian/patches/series
@@ -6,4 +6,4 @@ makefile
 vconsole
 microcode
 plymouth
-initrd-not-initramfs.patch
+initrd.img
-- 
2.39.1

